ArjunaJTS/jts/classes/com/arjuna/ats/jts/common/JTSEnvironmentBean.java

```diff
@@ -55,6 +55,15 @@
     @FullPropertyName(name = "com.arjuna.ats.jts.recovery.commitTransactionRetryLimit")
     private volatile int commitedTransactionRetryLimit = 3;
 
+    private volatile int transactionServiceId = 0xDEADBEEF;
```

Comment:
The policy is to maintain backward compatibility between minor releases. TAG-25 says the format is major.minor.micro.Final. I think that implies that we should wait for narayana 6major.minor.micro.Final